### ISSUE TITLE
Fluor not functional after plugging Mac keyboard in

### ISSUE THREAD

**Issue by joexcm** [Pyroh/Fluor#19]
Hi, amazing app, works perfectly for my Pro Tools needs.

When I add my official Apple wired keyboard (has all the relevant F keys up top), the F-keys don't function either way till I flip the default mode switch in the app and then it's all fine again. Wondered if you might be able to squish this little bug.

Many thanks for your work.

**Comment by Pyroh** [comment]
Could you please be more specific ? Describe what you do exactly, what you expect it to do and what it actually does. (No need to write a lot, don't worry).

**Comment by joexcm** [comment]
So I've just solved it but I'll keep my explanation of what the issue was in case it would be useful for an FAQ or anything.


- I've got one of [these](https://www.apple.com/uk/shop/product/MB110B/B/apple-keyboard-with-numeric-keypad-british) for my Macbook Pro, and in the Fluor settings I ask for the Apple function, not function key behaviour.
- Without the USB keyboard plugged in, Fluor functions as normal.
- However, if I connect the USB keyboard, this functionality flips so that the default setting for both laptop keyboard and USB keyboard is function key. This change of setting is not reflected in the GUI.
- As a side note, if I have function key as the default, inserting the keyboard does not flip this setting, it remains as programmed.

The solution was to go into keyboard system preferences and untick 'use all F1, F2 etc. key as standard function keys'. I think because I ticked this when the USB keyboard was plugged in it didn't affect the laptop's function keys, which then Fluor overrode anyway, but when the keyboard was plugged in again, the system preferences one-upped it.
Either way problem solved!
